src/components/Wrapper.js

```diff
@@ -3,7 +3,7 @@ import styled, { css } from 'styled-components/native';
 import { normalize } from '../config';
 
 export const Wrapper = styled.View`
-  padding: ${normalize(14)}px;
+  padding: ${normalize(16)}px;
```

Comment:
the padding for the filter section is not set correctly.

|||
|--|--|
![Simulator Screenshot - iPhone 15 Pro Max - 2024-07-29 at 09 39 52](https://github.com/user-attachments/assets/31e6d36c-e8f2-4a69-86c8-79f124b60d73)|![Simulator Screenshot - iPhone 15 Pro Max - 2024-07-29 at 09 39 58](https://github.com/user-attachments/assets/4cb4a15f-2615-45c5-9edc-f0d05d3c76da)

Reply:
will be fixed to

![Simulator Screenshot - iPhone 15 Plus - 2024-09-20 at 17 08 09](https://github.com/user-attachments/assets/6f76a18e-2336-495a-9c3b-c8e8a72e3d10)